src/cookie.mjs

```diff
@@ -40,7 +40,7 @@ export default class Cookie {
         // if it is invalid an error will be raised
         const parsedURL = new url.URL(requestURL);
 
-        const splitted = str.split("; ");
+        const splitted = str.split(/;\s*|;/);
```

Comment:
The `*` quantifier already matches zero occurances of `\s`, so the alternative `;` without `\s` is redundant. Can you simplify this to just `;\s*`?

Reply:
Done